bash_completion.sh

```diff
@@ -159,14 +167,22 @@ _buildtest ()
       ;;
 
     report|rt)
-      local opts="--end --failure --filter --format --help --helpfilter --helpformat --latest --no-header --oldest --pager --passed --start --terse  -e -f -h -n -p -s -t clear list summary"
-      COMPREPLY=( $( compgen -W "$opts" -- $cur ) )
-      case "${COMP_WORDS[2]}" in summary)
+      local opts="--color --end --failure --filter --format --help --helpfilter --helpformat --latest --no-header --oldest --pager --passed --start --terse  -e -f -h -n -p -s -t clear list summary"     
+      local copts=$(python -c "from rich.color import ANSI_COLOR_NAMES;print(' '.join(list(ANSI_COLOR_NAMES.keys())))")
+      arr1=($copts)
+      arr2=($opts)
```

Comment:
i dont see where `arr2` is being used can we remove this. I can see `$opts` is used later down the code

Reply:
Removed both arr1 and arr2